src/application/controllers/api/v3/Appointmentsv3.php

```diff
@@ -36,71 +36,63 @@ public function __construct()
         $this->attachments_parser = new \EA\Engine\Api\V2\Parsers\AttachmentsV2;
     }
     
-    public function get() {
-        
-        try {
-            $idUserIntegrated = $this->input->get('id_user_integrated');
-            $idPatientIntegrated = $this->input->get('id_patient_integrated');
-            $idServiceIntegrated = $this->input->get('id_service_integrated');
-            $page = $this->input->get('page');
-            $size = $this->input->get('length');
-            $sort = $this->input->get('sort');
-            $startDate = $this->input->get('startDate');
-            $endDate = $this->input->get('endDate');
+    // public function get() {
```

Comment:
Sorry but why do you comment all these code ?
Do you test all the case in this function after comment out?

Reply:
uncomment it, maybe my mistaken